src/test/java/org/ays/payload/Location.java

```diff
@@ -10,4 +11,26 @@ public class Location {
     private double latitude;
     private double longitude;
 
+    public static Location generate(double latitude, double longitude){
+        Location location = new Location();
+        location.setLatitude(latitude);
+        location.setLongitude(longitude);
+        return location;
+    }
+    public static Location generateLocation(double minLat, double maxLat, double minLon, double maxLon) {
+
+        double latitude = AysRandomUtil.generateLatitude(minLat,maxLat);
+        double longitude = AysRandomUtil.generateLongitude(minLon,maxLon);
+        return Location.generate(latitude,longitude);
+    }
+
+    public static Location generateLocationTR() {
```

Comment:
Metoda baktığımda Türkiye lokasyonuna oluşturmuyor.
```suggestion
    public static Location generate() {
```

Eğer Türkiye lokasyonuna oluşturuyorsa;
```suggestion
    public static Location generateForTurkey() {
```